src/App.js

```diff
@@ -1,7 +1,28 @@
+import { useEffect, useState } from 'react';
+import { fetchAllArticles } from './api';
 import './App.css';
+import Header from './components/Header';
+import Home from './components/Home';
+import Nav from './components/Nav';
+import { Route, Routes } from 'react-router-dom';
 
 function App() {
-  return <div className="App"></div>;
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    fetchAllArticles().then((data) => {
```

Comment:
Please use a 'loading' state to let the user know what's happening which doing the API request.